code/modules/projectiles/ammo_datums.dm

```diff
@@ -1544,12 +1544,10 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 */
 /datum/ammo/xeno
 	icon_state = "neurotoxin"
-	ping = "ping_x"
 	damage_type = TOX
 	flags_ammo_behavior = AMMO_XENO
 	var/added_spit_delay = 0 //used to make cooldown of the different spits vary.
 	var/spit_cost = 5
-	armor_type = "bio"
```

Comment:
Add these two back, but remove them from `/datum/ammo/xeno/larva_jelly`, since it inherits.